spec/Section 6 -- Execution.md

```diff
@@ -310,6 +336,32 @@ Unsubscribe(responseStream):
 
   * Cancel {responseStream}
 
+## Yield Subsequent Payloads
+
+If an operation contains subsequent payload records resulting from `@stream` or
+`@defer` directives, the {YieldSubsequentPayloads} algorithm defines how the
+payloads should be processed.
+
+YieldSubsequentPayloads(subsequentPayloads):
+
+  * While {subsequentPayloads} is not empty:
+  * Let {record} be the first complete item in {subsequentPayloads}.
+    * Remove {record} from {subsequentPayloads}.
+    * If {record} is a Deferred Fragment Record:
+      * Let {payload} be the result of running {ResolveDeferredFragmentRecord(record, variableValues, subsequentPayloads)}.
+    * If {record} is a Stream Record:
+      * Let {payload} be the result of running {ResolveStreamRecord(record, variableValues, subsequentPayloads)}.
+    * If {payload} is {null}:
```

Comment:
`payload` may be undefined here, if record is neither a stream or deferred fragment record?

Reply:
Each `record` in `subsequentPayloads` has to be either a stream record or a deferred fragment record.

Reply:
Currently, yeah. Should we put an assert here to enforce that so that future spec edits don't accidentally break this assumption?